packages/frontend-api/src/Model/Resolver/Settings/DisplayTimezoneQuery.php

```diff
@@ -22,6 +22,6 @@ public function __construct(
      */
     public function displayTimezoneQuery(): string
     {
-        return $this->displayTimeZoneProvider->getDisplayTimeZone()->getName();
+        return $this->domain->getDateTimeZone()->getName();
```

Comment:
Shouldn't it be the other way around? I would expect the display timezone provider to take into account the timezone settings in domains

Reply:
and 733dafcc0 :slightly_smiling_face: